### DISCUSSION TITLE
how to duplicate 1000 records and change their id?

### DISCUSSION THREAD

**Discussion by robokonk** [prisma/prisma#12753] [Q&A]
I using since 2 days prisma, is great.
I have one problem, how I can duplicate in prisma my base data in db:
```
[
  {"id":"6a9122b2-363b-4a4f-ad26-d6c55b51baed","title":"TailwindCSS"},
  {"id":"2ea8cfb0-44a3-4c07-bdc2-31ffa135ea78","title":"Apollo GraphQL"}
]
```

I want duplicate this rows x 1000, and changed id for each records as indyvidual id.
What I shoudl to do?

**Comment by austincrim** [comment]
Hey @robokonk 👋,

If you are using Prisma to autogenerate your IDs, you could write a Node script to insert those rows with just the `title` and Prisma will fill in the `id` field. If this is a pattern you would like to repeat, I would recommend checking out our [integrated seeding](https://www.prisma.io/docs/guides/database/seed-database) functionality.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
